File: L3/java 3/Hexagon.java
public class Hexagon extends Figure {

    private int side;

    Hexagon(int side) {
        this.side = side;
    }

    public double area() {
        return 3*side*side*Math.sqrt(3)/2;
    }

    public double perimeter() {
        return 6*side;
    }
}

File: L3/cpp 3/src/hexagon.cpp
#include <hexagon.hpp>
#include <cmath>


        Hexagon::Hexagon(int side) {
            this->side = side;
        }

        double Hexagon::area() {
            return 3*side*side*sqrt(3)/2;
        }

        double Hexagon::perimeter() {
            return 6*side;
        }